data_ingestion/04_Ingest_data_with_EMR.ipynb

```diff
@@ -0,0 +1,237 @@
+{
```

Comment:
<p>Update with links to videos</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1602/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>